kytos/core/rest_api.py

```diff
@@ -99,53 +91,3 @@ def render(self, content) -> bytes:
             separators=(",", ":"),
             default=_json_serializer,
         ).encode("utf-8")
-
-
-# pylint: disable=super-init-not-called
-class AStarletteOpenAPIRequest(_StarletteOpenAPIRequest):
-    """Async StarletteOpenAPIRequest."""
-
-    def __init__(self, request: Request, body: bytes) -> None:
-        """Constructor of AsycnStarletteOpenAPIRequest.
-
-        This constructor doesn't call super().__init__() to keep it async
-        """
-        self.request = request
-        self.parameters = RequestParameters(
-            query=self.request.query_params,
-            header=self.request.headers,
-            cookie=self.request.cookies,
-        )
-        self._body = body
-
-    @property
-    def body(self) -> Optional[str]:
-        body = self._body
-        if body is None:
-            return None
-        return body.decode("utf-8")
-
-
-# pylint: disable=super-init-not-called
-class StarletteOpenAPIRequest(_StarletteOpenAPIRequest):
-    """Sync StarletteOpenAPIRequest."""
-
-    def __init__(self, request: Request, body: bytes) -> None:
-        """Constructor of AsycnStarletteOpenAPIRequest.
-
-        This constructor doesn't call super().__init__() to keep it async
-        """
-        self.request = request
-        self.parameters = RequestParameters(
-            query=self.request.query_params,
-            header=self.request.headers,
-            cookie=self.request.cookies,
-        )
-        self._body = body
-
-    @property
-    def body(self) -> Optional[str]:
-        body = self._body
-        if body is None:
-            return None
-        return body.decode("utf-8")
```

Comment:
Deleted these methods since body no longer is obtained from event loop. [New](https://github.com/python-openapi/openapi-core/blob/master/openapi_core/contrib/starlette/requests.py), [old](https://github.com/python-openapi/openapi-core/blob/0.16.6/openapi_core/contrib/starlette/requests.py).